addons/payment_adyen/i18n/af.po

```diff
@@ -55,7 +55,7 @@ msgstr ""
 
 #. module: payment_adyen
 #: model:ir.ui.view,arch_db:payment_adyen.acquirer_form_adyen
-msgid "How to configure your Adyen account?"
+msgid "How to configure my Adyen account?"
```

Comment:
We don't modify the .po and .pot file in master.

We just modify the .pot in a stable release. Please remove all the modifications in theses files.